test/run-tests

```diff
@@ -622,6 +624,13 @@ def handle_task(gh, args, config, task, dry_run=False):
     status_context = f"{config['name']}/{task.image['name']}"
 
     if not dry_run:
+        logging.info(
+            "Claiming task %s/%s/%d:%s",
```

Comment:
Maybe consider adding a `__str__`  method to Tasks, so that you can just use `logging.info("Claiming %s", task)` instead of repeating the details in each log call. `___str___()` could then return something like `task linux-system-roles/network/123:Fedora`

Reply:
Good idea, will do.